src/main/java/no/nav/brukernotifikasjon/schemas/builders/util/ValidationUtil.java

```diff
@@ -154,4 +119,39 @@ public static String validateMaxLength(String field, String fieldName, int maxLe
         }
         return field;
     }
+
+    public static <T> String validateLinkAndConvertToString(T field, String fieldName, int maxLength, boolean required) {
```

Comment:
Kunne denne metoden heller returnert en URL? Det er en datatype som man vet at er en gyldig URL. Hvis det er behov for en URL som en streng et annet sted, så kan man heller kalle toString på URL-en? Eller er det noe jeg ikke har fått med meg her? 🙂

Reply:
Skal jeg returnere null når link er en tom String? Også ha en sjekk på null [her](https://github.com/navikt/brukernotifikasjon-schemas/blob/b9c8c40b98d1b9cdd2edda689b0659dc35945f21/src/main/java/no/nav/brukernotifikasjon/schemas/builders/BeskjedBuilder.java#L68) og i BNB, så ikke toString feiler? 
Eller hvordan er det best å håndtere at beskjed kan være en tom String, men oppgave kan ikke være det? 😊  Eller tenker jeg feil nå? 🤔